php/libraries/Utility.class.inc

```diff
@@ -824,6 +824,35 @@ class Utility
         return $test_names;
     }
 
+    /**
```

Comment:
Why is this PR adding this function?

Other than that it looks good, given that it's just changing comments and still passing Travis..

Reply:
Rebase error I guess. I'll update it to remove this and also change the base branch.